web/ui-go/go.sum

```diff
@@ -0,0 +1,20 @@
+github.com/golang/protobuf v1.5.0/go.mod h1:FsONVRAS9T7sI+LIUmWTfcYkHO4aIWwzhcaSAoJOfIk=
```

Comment:
As above - I suspect that with a Bazel `BUILD` this `go.sum` may be able to be removed?